app/src/main/res/values/strings.xml

```diff
@@ -45,6 +45,8 @@
   <string name="Sha">SHA</string>
   <string name="Staging">Staging</string>
   <string name="Submit_feedback">Submit feedback</string>
+  <string name="token_reward_surveys">Reward_Surveys</string>
+  <string name="token_reward_survey_count">reward_survey_count</string>
```

Comment:
strings in this file shouldn't be hard coded--let's chat about this, think we should to review strings